bb/ramfs.go

```diff
@@ -115,23 +114,29 @@ func ramfs() {
 
 	paths[filepath.Join(config.Gopath, "src/github.com/u-root/u-root/bb/bbsh")] = []string{"init", "ubin"}
 
-	if *extraPaths != "" {
-		extras := strings.Split(strings.TrimSpace(*extraPaths), " ")
-		for _, x := range extras {
-			p := strings.Split(x, ":")
-			if len(p) != 2 {
-				p = append([]string{"/"}, p...)
+	if *extraCmds != "" {
+		copyc := strings.Fields(*extraCmds)
+		for _, eachPath := range copyc {
+			// eachPath must be an absolute filepath for os.Stat
```

Comment:
That's not really right. It can be  a path relative to cwd. I think you can remove this comment.